README.md

```diff
@@ -48,6 +48,15 @@ To run this:
 
 `$ node ./src/bin/download-raw.js`
 
+### Validate Raw Data
+
+This will validate the raw data using the standard https://github.com/openactive/data-model-validator library
+and save the results back in the database in the`raw_data` table. 
+
+To run this:
+
+`$ node ./src/bin/validate-raw-data.js`
+
```

Comment:
We could really use npm run scripts, the advantage of those is that you avoid encoding the filename/path in lots of places. Though perhaps this is the only place anyway.

Reply:
https://github.com/openactive/conformance-services/issues/27

Reply:
When we set up Heroku some more hard coded paths will appear so this is a good point